docs/sources/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -19,96 +19,112 @@ you can get event description and event tags. The text field can include links t
 
 {{< figure src="/static/img/docs/v46/annotations.png" max-width="800px" >}}
 
-## Native annotations
+You can annotate panels in three ways:
 
-Grafana comes with a native annotation store and the ability to add annotation events directly from the graph panel or via the [HTTP API]({{< relref "../../../developers/http_api/annotations/" >}}).
+- Directly in the panel, using the built-in annotations query (add link)
+- Querying other data sources, using the dashboard settings (add link)
+- Using the HTTP API
 
-### Add annotation
+This page explains the first two options; for information about using the HTTP API, refer to [Annotations API]({{< relref "../../../developers/http_api/annotations/" >}}).
+
+## Add annotations in panels
+
+Grafana comes with a native annotation store and the ability to add annotation events directly from the graph panel using the built-in annotation query(link here) that exists on all dashboards.
+
+### Add an annotation
 
 1. In the dashboard click on the Time series panel. A context menu will appear.
 1. In the context menu click on **Add annotation**.
    ![Add annotation context menu](/static/img/docs/time-series-panel/time-series-annotations-context-menu.png)
 1. Add an annotation description and tags(optional).
-   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-annotation.png)
 1. Click save.
 
 Alternatively, to add an annotation, Ctrl/Cmd+Click on the Time series panel and the Add annotation popover will appear
 
-### Add region annotation
+### Add a region annotation
 
 1. In the dashboard Ctrl/Cmd+click and drag on the Time series panel.
    ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-region-annotation.gif)
 1. Add an annotation description and tags(optional).
 1. Click save.
 
-### Edit annotation
+### Edit an annotation
 
 1. In the dashboard hover over an annotation indicator on the Time series panel.
    <!--![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-edit-annotation.gif)-->
 1. Click on the pencil icon in the annotation tooltip.
 1. Modify the description and/or tags.
 1. Click save.
 
-### Delete annotation
+### Delete an annotation
 
 1. In the dashboard hover over an annotation indicator on the Time series panel.
    <!--![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-edit-annotation.gif)-->
 1. Click on the trash icon in the annotation tooltip.
 
-### Built-in query
+## Add annotations through dashboard settings
```

Comment:
```suggestion
## Add annotations queries in the dashboard settings
```

Reply:
In other places we mention the **Annotation settings**, which are accessible through **Dashboard settings**.
I would like to distinguish between **Create** and **Fetch**. Sometimes we refer to **Add** to create new annotations and sometimes we use it to **Fetch** existent annotations from other places. Not sure how to do it though

Reply:
What about this? I've also changed the previous major heading to "Create annotations in the panel" to distinguish the two actions.
```suggestion
## Fetch annotations through dashboard settings
```